src/problem-analysis/state-of-the-art/SOTA.bib

```diff
@@ -6,3 +6,48 @@ @report{alpha-go-zero
     year = {2018},
     pages = {32},
 }
+
+@misc{chess-com-chess-engines,
```

Comment:
@SebastianMygind Try to use different entry types than just `@misc`: https://www.overleaf.com/learn/latex/Bibliography_management_with_biblatex#Reference_guide

Reply:
I understand, will begin to from this point forward, it would just be a lot of work to go through all the sources that are already used.